src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -708,4 +708,11 @@ Valid from:</comment>
 2 - package source url
 3 - Log Message</comment>
   </data>
+  <data name="VerifyCertTrustOfflineWhileRevocationModeOffline" xml:space="preserve">
+    <value>The revocation function was unable to check revocation because the certificate is not available in the cached certificate revocation list and certificateRevocationMode has been set to offline. For more information, visit https://aka.ms/certificateRevocationMode.</value>
```

Comment:
Do we want to say the revocation function or does something makes sense? 

We were unable to do a revocation check...

Reply:
//cc @karann-msft Why mention function?

Reply:
@nkolev92 that's what comes from the API.